src/test/java/com/minecolonies/test/AbstractTest.java

```diff
@@ -0,0 +1,40 @@
+package com.minecolonies.test;
+
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.util.LanguageHandler;
+import com.minecolonies.util.Log;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import static org.mockito.Matchers.anyObject;
+import static org.mockito.Matchers.anyString;
+import static org.powermock.api.mockito.PowerMockito.*;
+
+/**
+ * Abstract test class to abstract away some common uses functionality in Tests
+ */
+@PrepareForTest({ColonyManager.class, LanguageHandler.class, Log.class})
+@PowerMockIgnore("javax.management.*")
```

Comment:
I didn't know this was a thing, we should just ignore the Log class, and then everything can go back to normal (the way we had it). Also I feel like the ColonyManger and LanguageHandler stuff doesn't need to be here, LanguageHandler could be I guess.